### ISSUE TITLE
Not working on PrestaShop 1.5.6.1

### ISSUE THREAD

**Issue by maziq** [soulseekah/prestashop-variable-shipping#4]
Hello,

I install plugin and I don't see field to edit shipping price. I'm using Polish language:

![123456](https://cloud.githubusercontent.com/assets/7867978/3358144/8755889e-fada-11e3-9c19-2b8fdcf50ab8.png)

**Comment by soulseekah** [comment]
Make sure you've configured the carrier correctly by adding it and whatnot. I forgot the exact procedure of how to install it.

**Comment by clintallan** [comment]
As I remember if you make any modifications to the carrier it doesn't work. You install the module and then when you are creating a manual order in the back office under the shipping section you have an option to set shipping price. You also get option to select other carriers. 

If you edit the variable shipping carrier the module doesn't work.

**Comment by soulseekah** [comment]
@clintovsina thanks for your response. @maziq does this make sense? Did it work?

**Comment by patriziavergassola** [comment]
Hello,
I also use Prestashop 1.5.6.1 and it doesn't work.
When I try to update the shipping costs, nothing happens.
Patrizia

**Comment by patriziavergassola** [comment]
It indeed works, but it doesn't when the shipping free and, on the contrary, I want to add shipping fees.
Is there a way to change that?

**Comment by soulseekah** [comment]
@patriziavergassola you'll have to be more clear and provide more debugging information. Can you send me working administration credentials so I can test it out on your setup? My e-mail is gennady[at]kovshenin[dot]com

**Comment by patriziavergassola** [comment]
Hello,

I have a Prestashop store: any order above the €50 is set to have a free shipping. 

Sometimes I want to add an order from the administration panel and, even if is above the € 50, I want the Client to pay for the shipping fees.

Currently your module doesn’t do that.

I hope I was clear.

Patrizia

Dott.ssa Patrizia Vergassola / I-NAT

patrizia.vergassola@i-nat.it mailto:patrizia.vergassola@i-nat.it 

---

I-NAT SOLUTIONS SNC

Corso Cavour 111
19121 La Spezia (SP)
Tel. +39 0187 520 168 
Fax +39 0187 524 278
skype: patriziavergassola

site www.i-nat.it http://www.i-nat.it/ 

Il testo e gli eventuali documenti trasmessi contengono informazioni riservate al destinatario indicato. La seguente e-mail è confidenziale e la sua riservatezza è tutelata legalmente dal codice della privacy art.13 del D.Lgl. 30.6.2003 n°196. La lettura, copia od altro uso non autorizzato o qualsiasi altra azione derivante dalla conoscenza di queste informazioni sono rigorosamente vietate. Se pensi di non essere il destinatario di questa mail, o se hai ricevuto questa mail per errore, sei pregato di darne immediata comunicazione al mittente e di provvedere immediatamente alla sua distruzione. Questo messaggio rappresenta l'opinione del mittente e non necessariamente l'opinione della società.

Da: Gennady Kovshenin [mailto:notifications@github.com] 
Inviato: mercoledì 17 dicembre 2014 15.49
A: soulseekah/prestashop-variable-shipping
Cc: patriziavergassola
Oggetto: Re: [prestashop-variable-shipping] Not working on PrestaShop 1.5.6.1 (#4)

@patriziavergassola https://github.com/patriziavergassola  you'll have to be more clear and provide more debugging information. Can you send me working administration credentials so I can test it out on your setup? My e-mail is gennady[at]kovshenin[dot]com

**Comment by soulseekah** [comment]
Makes sense. I will think about it and look into it if I have time soon.
Thanks for reporting.

## 

Regards,
Gennady Kovshenin

2014-12-17 20:07 GMT+05:00 patriziavergassola notifications@github.com:
